docs/porting.md

```diff
@@ -383,13 +383,20 @@ Of course this introduces a scaling bottleneck when `SafeCounter` instances are
 concurrently updated. It's possible to implement more optimized locking
 strategies, but doing so requires knowledge of the problem.
 
+### Do I need to thread everything?
+Before you thread your code, you might consider whether another
+(possibly decideable) pattern like Message Passing with vector clocks
+could solve your problem without handling threads at all; 
+with task isolation, distributed over multiple processes on multiple nodes.
```

Comment:
We don't want this section, this isn't really relevant to the content on this website, and will notbe helpful to the average reader without a lot more context that also doesn't fit here.